archinfo/arch_soot.py

```diff
@@ -1,12 +1,17 @@
 
 from .arch import Arch, register_arch, Endness
+import re
+
+import logging
+l = logging.getLogger('archinfo.arch_soot')
 
 
 class SootMethodDescriptor(object):
-    def __init__(self, class_name, name, params):
```

Comment:
A general thought: This is a good chance to add `__slots__` to all these descriptors so it's faster to create them in CPython (and saves some memory).